moderatorFrontend/src/main/kotlin/views/users/AddUser.kt

```diff
@@ -85,13 +86,13 @@ class AddUser(props: AddUserProps) : RComponent<AddUserProps, AddUserState>(prop
     val response = NetworkManager.post<String>(
       url = "$apiBase/user/edit",
       body = EditUserData(
-          userId = (props.config as Config.Edit).user.id,
-          name = state.userNameTextFieldValue.emptyToNull(),
-          password = state.userPasswordTextFieldValue.emptyToNull(),
-          permissions = state.userPermissions
-            .takeIf { (props.config as Config.Edit).user.permissions != it }
-            ?.map { it.name }
-        )
+        userId = (props.config as Config.Edit).user.id,
+        name = state.userNameTextFieldValue.emptyToNull(),
+        password = state.userPasswordTextFieldValue.emptyToNull(),
+        permissions = state.userPermissions
+          .takeIf { (props.config as Config.Edit).user.permissions != it }
+          ?.map { it.name }
+      )
```

Comment:
Not sure if the left or right formatting is correct. Have you set the kotlin code style and 2 - 2 - 4 indent settings? If you're not sure what I mean by this, ask and I'll show you.

Reply:
Code style settings are correct 👍